DEVELOPMENT-GUIDE.md

```diff
@@ -140,18 +140,18 @@ Each layer from highest level (Java) down:
 Once a [Java Exception](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/design.html#java_exceptions) has been thrown, there are exceedingly few JNI operations a caller is allowed to perform.
 What's more, it is easy to not notice that there is a Java exception pending.
 So long as you do not interact with `JNIEnv`/`raii_env` directly, you shouldn't need to worry about Java exceptions as the ACCP objects/methods which use them already have appropriate checks which throw C++ exceptions (specifically `java_ex`) instead.
-If you need to throw an exception, it should almost always be a C++ exception and thrown using the `throw_java_ex` or `throw_openssl` methods. (The former is more efficient but the latter MUST be used when the exception is due to an error state reported by OpenSSL.)
```

Comment:
Let's add a note to this page that as ACCP was originally built on openssl, many functions/etc. refer to openssl when they really mean any implementation of `libcrypto` such as AWS-LC.

Reply:
good call, will add.